### PR TITLE
Add 2-FA instructions for non-technical users

### PR DESCRIPTION
**Pull request by koboldunderlord**
2-FA readme, should be easy to read and simplifies access to users as much as possible

### CODE REVIEW

File: 2FA.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+5. Open the "Authenticator" app
+6. Press the + button in the bottom right-hand corner
+7. Authenticator should prompt you for access to your camera- hit "OK"
+8. Line up the black and white QR code with the target box that appears on your camera
+9. Now, whenever you log in to Mastodon, open the Authenticator app and enter the 6 digit code that appears above the "mastodon.social (email address)" text
+
+# Disabling 2-factor Authentication
+
+1. Go to [the 2-factor authentication settings page](https://mastodon.social/settings/two_factor_auth)
+2. Press the big blue "Disable" button underneath your QR code ![disable button screenshot](http://imgur.com/Om61dvs.jpeg)
```
Comment (ineffyble) [inline 2FA.md]: Do we want to use an image hosted on imgur?

File: 2FA.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+4. Download the above application on your phone
+5. Open the "Authenticator" app
+6. Press the + button in the bottom right-hand corner
+7. Press "Scan a barcode"
+8. Line up the black and white QR code with the target box that appears on your camera
+9. Now, whenever you log in to Mastodon, open the Authenticator app and enter the 6 digit code that appears above the "mastodon.social (email address)" text
+
+## iPhone
+
+__Recommended Application:__ iPhone: [Authenticator by Matt
```
Comment (ineffyble) [inline 2FA.md]: Curious why you've recommended this over the Google Authenticator app

File: 2FA.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# 2-Factor Authentication
+
+2-Factor Authentication is a security mechanism that requires you to enter a computer generated code every time you log into Mastodon.
```
Comment (ineffyble) [inline 2FA.md]: "computer generated code from your phone" maybe?

Comment (vahnj) [inline 2FA.md]: Good clarification imo, will change that

**Pull request closed without merging**